extensions/wikia/Wall/WallHistoryController.class.php

```diff
@@ -161,9 +161,6 @@ private function getHistoryMessagesArray() {
 	}
 
 	private function getFormatedHistoryData( $history, $threadId = 0 ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The Cyclomatic Complexity of this function "getFormatedHistoryData" is 21 which is greater than 20 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1541)